keps/sig-cli/3104-introduce-kuberc/README.md

```diff
@@ -305,21 +313,32 @@ required) or even code snippets. If there's any ambiguity about HOW your
 proposal will be implemented, this is the place to discuss them.
 -->
 
-During alpha this feature will be enabled through the `KUBECTL_KUBERC=true` environment variable. The file will default to being located in `~/.kube/kuberc`. A flag will allow overriding this default location with a path i.e. `kubectl --kuberc /var/kube/rc`.
+For beta this feature will be enabled by default. However users can disable it
+setting the `KUBECTL_KUBERC` environment variable to `false`.
 
-Three initial top level keys are proposed.
+By default, the configuration file will be located in `~/.kube/kuberc`. A flag
+will allow overriding this default location with a specific path, for example:
+`kubectl --kuberc /var/kube/rc`.
 
-* `apiVersion` to determine the version of the config.
-* `kind` to keep consistency with Kubernetes resources.
-* `aliases` for users to declare their own aliases for commands with flags and values.
-* `overrides` for users to set default flags to apply to commands.
+The following top-level keys are proposed, alongside the kubernetes `metav1.TypeMeta`
+fields (`apiVersion`, `kind`):
 
-`aliases` will not be allowed to override builtins but take precedence of plugins i.e. builtins -> aliases -> plugins. Additional flags and values will be appended to the end of the aliased command. It is the responsibility of the user to define aliases with this in mind.
+* `aliases` Allows users to declare their own command aliases, including flags and values.
+* `overrides` Enables users to set default flags to be applied to commands.
 
-`overrides` is modeled after all configurable behavior being implemented as flags first. This is a design decision that was made after modeling out the intended behavior and realizing that targeting flags filled the use cases. A merge will be done in the execution of the command for flags with inline overrides taking precedence.
+`aliases` will not be permitted to override built-in commands but will take
+precedence over plugins (builtins -> aliases -> plugins). Any additional flags
+and values will be appended to the end of the aliased command. Users are
+responsible for defining aliases with this behavior in mind.
+
+`overrides` is designed based on the principle that all configurable behavior is
+initially implemented as flags. This design decision was made after analyzing the
+intended behavior and realizing that targeting flags effectively addresses the
+use cases. During command execution, a merge will be occur, with inline overrides
+taking precedence over the defaults.
 
 ```
-apiVersion: kubectl.config.k8s.io/v1alpha1
+apiVersion: kubectl.config.k8s.io/v1beta1
```

Comment:
cc: @liggitt

Reply:
was there a question for me here, or was this just FYI?